index.html

```diff
@@ -240,5 +240,34 @@ <h3>Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="form-container">
+         <div class="form-text">
+           <h3>Contact me</h3>
+         </div>  
+      <div class="paragragh">
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+      
+      <div class="contact-form">
+        <form action="https://formspree.io/f/xzboload" method="post" name="form-collection" class="my-form-collection">
+          <ul>
+            <li>
+                <input type="text" placeholder="Yeremias" maxlength="30" required>
+            </li>
+  
+            <li>
+                <input type="email" placeholder="natajayanj14@gmail.com" required>
+            </li>
+  
+            <li>
+                <textarea name="text" cols="30" rows="10" placeholder="write your massage here" maxlength="500"
+                required></textarea>
+            </li>
+          </ul>
+          <button type="submit" class="contact-button">Get in touch</button>
```

Comment:
You did not address the previous reviewer's requirements about the background image. Please do so.